tests/Foliated_triangulation_4_test.cpp

```
@@ -0,0 +1,126 @@
+      -7, seeded.counts(), {});
+  CHECK_EQ(clamped.timeslices(), 2);
+  CHECK_EQ(clamped.spatial_volume_profile().size(), 2);
+
+  auto counts = S4Counts{1, 2, 3, 4, 5, 1, 2, 1, 1};
+  auto abstract = FoliatedTriangulation4::from_counts_for_validation(
+      2, counts, FoliatedTriangulation4::Profile{1, 1});
+  auto const inventory = abstract.proposal_inventory();
+  CHECK_EQ(inventory.spatial_tetrahedra, 4);
+  CHECK_EQ(inventory.timelike_edges, 2);
+  CHECK_EQ(inventory.mixed_triangles, 3);
+  CHECK_EQ(inventory.timelike_tetrahedra, 4);
```

Cppcheck (2.21.0)

[style] The function 'as_move_4d' is never used.

(unusedFunction)